src/arrayMethodJoin.js

```diff
@@ -4,8 +4,23 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
+  [].__proto__.join2 = function(SEPARATOR = ',') {
     // write code here
+    let RESULT = '';
+
+    for (let i = 0; i < this.length; i++) {
+      const CURRENT_NUMBER = this[i];
```

Comment:
the current number is not the perfect name here - after all you can have an array of anything, not only numbers

In addition, see my comment on naming at the top.